sdk/communication/communication-short-codes/package.json

```diff
@@ -60,7 +60,7 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
-    "@azure/communication-common": "^2.1.0",
+    "@azure/communication-common": "^2.0.0",
```

Comment:
Any reason for the downgrade?

Reply:
This version isn't released yet, and we're in a short timeline, so the team suggested downgrading it